internal/blocksync/pool.go

```diff
@@ -273,7 +257,8 @@ func (pool *BlockPool) AddBlock(peerID p2p.ID, block *types.Block, extCommit *ty
 		if diff < 0 {
 			diff *= -1
 		}
-		if diff > maxDiffBetweenCurrentAndReceivedBlockHeight {
+		const maxDiff = 100 // maximum difference between current and received block height
```

Comment:
Why is this constant?

Reply:
I've removed it in the other PR. Please ignore.